setup.cfg

```diff
@@ -1,2 +1,7 @@
 [bdist_wheel]
 universal=1
+
+[tool:pytest]
```

Comment:
FWIW I dislike putting a pytest config into a packaging-related file.

Reply:
it's pending pending deprecation https://github.com/pytest-dev/pytest/issues/3523

the pyproject.toml is preferred